dev-lang/luajit/luajit-2.1.0_beta2.recipe

```diff
@@ -0,0 +1,65 @@
+SUMMARY="A Just-In-Time Compiler (JIT) for the Lua programming language"
+DESCRIPTION="Lua is a powerful, dynamic and light-weight programming \
+language. It may be embedded or used as a general-purpose, stand-alone \
+language. LuaJIT is a Just-In-Time Compiler (JIT) for it."
+HOMEPAGE="http://luajit.org/luajit.html"
+COPYRIGHT="2005-2015 Mike Pall"
+LICENSE="MIT"
+REVISION="1"
+SOURCE_URI="http://luajit.org/download/LuaJIT-$portVersion.tar.gz"
+CHECKSUM_SHA256="713924ca034b9d99c84a0b7b701794c359ffb54f8e3aa2b84fad52d98384cf47"
+SOURCE_DIR="LuaJIT-2.1.0-beta2"
+PATCHES="luajit-$portVersion.patchset"
+
+ARCHITECTURES="x86 !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+PROVIDES="
+	luajit$secondaryArchSuffix = $portVersion compat >= 5.1
+	lib:libluajit_5.1$secondaryArchSuffix = $portVersion compat >= 2
+	cmd:luajit_$portVersion$secondaryArchSuffix
+	cmd:luajit$secondaryArchSuffix
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	luajit${secondaryArchSuffix}_devel = $portVersion compat >= 5.1
+	devel:libluajit_5.1$secondaryArchSuffix = $portVersion compat >= 2
+	"
+REQUIRES_devel="
+	luajit$secondaryArchSuffix == $portVersion base
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	"
+BUILD_PREREQUIRES="
+	cmd:make
+	cmd:gcc$secondaryArchSuffix
+	"
+
+BUILD()
+{
+	# Use amalg target as recommended on luajit homepage to get better performance
+	make $jobArgs amalg PREFIX=$prefix
+}
+
+INSTALL()
+{
+	make install PREFIX=$prefix INSTALL_LIB="$libDir" INSTALL_BIN="$binDir" \
+		INSTALL_INC="$includeDir" INSTALL_SHARE="$dataDir" \
+		INSTALL_MAN="$manDir/man1" INSTALL_LMOD="$dataDir/lua/5.1"
+
+	ln -s libluajit-5.1.so.2.1.0 $libDir/libluajit-5.1.so.2
```

Comment:
@waddlesplash, I do agree that it does not **_look_** right, but if you are creating a symlink, then it is OK to use:
```
ln -s relative/path/to/target $directory/symlink-name
```
because **`relative/path/to/target`** is _not_ relative to the **_current_** directory but to the _directory where the symlink is created_. That's why the following command:
```
ln -s libluajit-5.1.so.2.1.0 $libDir/libluajit-5.1.so.2
```
will create a «**`libluajit-5.1.so.2 -> libluajit-5.1.so.2.1.0`**» symlink in **`$libDir`**, no matter which current directory you are in when the command is executed.

(Of course, most users go to the directory where the symlink is to be created (before creating it) because this allows to use the sehll auto-complete for the **`relative/path/to/`**. This is why very few people know that "local" symlinks (I mean symlinks to files in the same directory) can be created even if you're not in the target dir.)

So @miqlas, you can safely use the command I had suggested earlier:
```
ln -s libluajit-5.1.so.2.1.0 $libDir/libluajit-5.1.so.2
```
For the record, you can get the same result with:
```
ln -s -r $libDir/libluajit-5.1.so.2.1.0 $libDir/libluajit-5.1.so.2
```
So you have the choice ;-)